src/tests/CMakeLists.txt

```diff
@@ -12,7 +12,8 @@ file(COPY ${MAM4XX_HAERO_DIR}/bin/test-launcher DESTINATION ${PROJECT_BINARY_DIR
 add_library(mam4xx_tests kohler_verification.cpp
                          atmosphere_utils.cpp
                          testing.cpp)
-target_include_directories(mam4xx_tests PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+# target_include_directories(mam4xx_tests PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(mam4xx_tests PUBLIC ${PROJECT_BINARY_DIR}/include)
```

Comment:
I wonder if this is even needed.

Reply:
15's gone, but my setup isn't finding the header if I don't include `include` 🙃